docs/source/quick_start.rst

```diff
@@ -0,0 +1,140 @@
+=============================
+Getting Started with Packetry
+=============================
+
+Install Cynthion
+----------------
+
+Before installing Packetry, follow the instructions to `install Cynthion host software <https://cynthion.readthedocs.io/en/latest/getting_started.html>`__ and then ensure that your Cynthion is up to date by running:
+
+.. code::
+
+   cynthion update
+
+
+Install Packetry
+----------------
+
+.. tab:: Linux
+
+    .. tab:: Binary
+
+        AppImage coming soon! For now you can build and install with Cargo.
+
+    .. tab:: Cargo
+
+        **Install Prerequisites**
+
+        Install the Rust build tools, other essential build tools, and GTK 4 headers:
+
+        .. tab:: Debian/Ubuntu
+
+            .. code::
+
+                apt install rustc cargo build-essential libgtk-4-dev git
+
+        .. tab:: Fedora
+
+            .. code::
+
+                yum install rust cargo make gcc gcc-c++ gtk4-devel pango-devel git
+
+        Note that Packetry requires a minimum Rust version of 1.74. If your distribution's packages are older than this, use `rustup <https://rustup.rs/>`__ to get the latest Rust toolchain and manage your Rust installation.
+
+        **Install Packetry**
+
+        Clone the Packetry repository:
+
+        .. code::
+
+            git clone https://github.com/greatscottgadgets/packetry.git
+
+        **Build Packetry**
+
+        .. code::
+
+            cd packetry
+            cargo build
+
+        Note: Do not build with ``--all-features``. All the optional features currently in the package are for debug/test purposes only, and will prevent or degrade normal use of the application if enabled. See ``Cargo.toml`` for details.
+
+.. tab:: macOS
+
+    .. tab:: Binary
+
+        Homebrew installation coming soon!
+
+    .. tab:: Cargo
+
+        **Install Prerequisites**
+
+        Use `rustup <https://rustup.rs/>`__ to install the latest Rust toolchain and manage your Rust installation.
+
+        Use `Homebrew <https://brew.sh/>`__ to install GTK 4:
+
+        .. code::
+
+            brew install gtk4
+
+        **Install Packetry**
+
+        Clone the Packetry repository:
+
+        .. code::
+
+            git clone https://github.com/greatscottgadgets/packetry.git
+
+        **Build Packetry**
+
+        .. code::
+
+            cd packetry
+            cargo build
+
+        Note: Do not build with ``--all-features``. All the optional features currently in the package are for debug/test purposes only, and will prevent or degrade normal use of the application if enabled. See ``Cargo.toml`` for details.
+
+.. tab:: Windows
+
+   Download and run the Windows installer from the `latest release <https://github.com/greatscottgadgets/packetry/releases/latest>`__.
```

Comment:
We should also give instructions for building from source on Windows. We want Windows-based developers to be able to contribute easily.

A suitable workflow would be something like:
```
# Get vcpkg and set it up.
git clone https://github.com/microsoft/vcpkg.git
cd vcpkg
bootstrap-vcpkg.bat

# This will take a while - it builds and installs GTK and all its dependencies from source.
vcpkg install gtk pkgconf

# Set environment variables so that Packetry can find its dependencies. These will only
# persist for the lifetime of the current command window, so you may want to set these
# up elsewhere for future use.
set PKG_CONFIG=%CD%\installed\x64-windows\tools\pkgconf\pkgconf.exe
set PKG_CONFIG_PATH=%CD%\installed\x64-windows\lib\pkgconfig
set PATH=%PATH%;%CD%\installed\x64-windows\bin
cd ..

# Get Packetry and build it.
git clone https://github.com/greatscottgadgets/packetry.git
cd packetry
cargo build --release
```